third_party/terraform/resources/resource_compute_router_nat.go

```diff
@@ -0,0 +1,385 @@
+package google
```

Comment:
Since this resource is only going to be available in the beta provider, you'll want to name it .go.erb and add some logic to make sure it ends up empty in TPG. Here's an example: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/utils/iam_compute_subnetwork.go.erb

Reply:
OK, I'll do this last since I've been directly copying out of MM into build/terraform-beta. Leaving this open for now.

Reply:
Done.